h/streamer/app.py

```diff
@@ -27,6 +27,7 @@ def create_app(_global_config, **settings):
     config.add_route("ws", "/ws")
     config.add_route("annotation", "/a/{id}", static=True)
     config.add_route("api.annotation", "/api/annotations/{id}", static=True)
+    config.add_route("activity.user_search", "/users/{username}", static=True)
```

Comment:
Based on the comment above and the other few routes we register here, I reckon this is all we need to do.

Reply:
It's worth mentioning the comment is from a commit from 2016 😅. Maybe we want to revisit this at some point.

Reply:
See the Pyramid docs, but as far as I know "static routes" like these are still the right way to generate URLs to external apps or sites.